components/SingleClassView/Attendance/AttendanceComponent.tsx

```diff
@@ -78,27 +93,33 @@ export const AttendanceComponent: React.FC<AttendanceComponentProps> = ({ classI
             />
           </LocalizationProvider>
         </div>
-        <div className={styles.missingAttendance}>
-          {loadMissingAttendance ? (
-            <CustomLoader></CustomLoader>
-          ) : (
-            <MissingAttendanceComponent
-              changeDate={changeDate}
-              missingAttendance={missingAttendance}
-            />
-          )}
-        </div>
+        {(user && (user.role == "Teacher" || user.role == "Admin")) && (
+           <div className={styles.missingAttendance}>
+           {loadMissingAttendance ? (
+             <CustomLoader></CustomLoader>
+           ) : (
+             <MissingAttendanceComponent
+               changeDate={changeDate}
+               missingAttendance={missingAttendance}
+             />
+           )}
+         </div>
```

Comment:
lets comment this ternary operator for what gets rendered based on the role

Reply:
I thought we only wanted missing attendance to be displayed for Teacher and Admins